orderly_web/__init__.py

```diff
@@ -4,10 +4,15 @@
 from orderly_web.start import start
 from orderly_web.status import status
 from orderly_web.stop import stop
+from orderly_web.users import add_users, add_groups, add_members, grant
 
 __all__ = [
     pull,
     start,
     status,
-    stop
+    stop,
+    add_users,
```

Comment:
do we need to export all of these? Could we not export the module instead?

Reply:
There's one private method in there